autoload/codefmt.vim

```diff
@@ -393,3 +393,21 @@ endfunction
 function! codefmt#DisableIsAvailableChecksForTesting() abort
   let s:check_formatters_available = 0
 endfunction
+
+
+""
+" @private
+" Undoes the effect of @function(#DisableIsAvailableChecksForTesting), allowing
+" FORMATTER.IsAvailable checks to work normally.
+function! codefmt#DontDisableIsAvailableChecksForTesting() abort
+  let s:check_formatters_available = 1
+endfunction
```

Comment:
Seems like one method that takes an argument would be more natural.  If you want to stick with two methods, you should avoid double negatives.  DontDisable is better written Enable